build.go

```diff
@@ -116,6 +122,13 @@ func Build(entries EntryResolver, dependencies DependencyService, clock chronos.
 			}
 		}
 
+		if val, ok := os.LookupEnv("BP_WEB_SERVER"); ok && val == "httpd" {
+			err = generateConfig.Generate(context.WorkingDir, context.Platform.Path)
```

Comment:
Do we need to check for `httpd.conf` here as well to avoid overwriting existing config options?

Reply:
Yeah. I intend for the setting of the variable to override any `httpd.conf` file that would  appear in the working directory.